src/te/schedule/auto_inline_elem_wise.cc

```diff
@@ -113,6 +117,8 @@ void AutoInlineInjective(Schedule sch) {
 
 TVM_REGISTER_GLOBAL("schedule.AutoInlineElemWise").set_body_typed(AutoInlineElemWise);
 
+TVM_REGISTER_GLOBAL("schedule.AutoInlineBroadcast").set_body_typed(AutoInlineBroadcast);
```

Comment:
Also add an interface to `tvm/te/schedule_pass.h`.

Reply:
> Also add an interface to `tvm/te/schedule_pass.h`.

Thanks! The API was added.